sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiSinkSchemaUpdateIT.java

```diff
@@ -131,7 +131,15 @@ public static Iterable<Object[]> data() {
   private static final int ORIGINAL_N = 60;
   // for dynamic destination test
   private static final int NUM_DESTINATIONS = 3;
-  private static final int TOTAL_NUM_STREAMS = 9;
```

Comment:
R: @ahmedabu98 who took care of SchemaUpdate tests

Reply:
@akashorabek can we set `TOTAL_NUM_STREAMS=6` and `SCHEMA_UPDATE_TRIGGER=2`? I worry we might be missing some unknown edge cases by restricting it too much

Reply:
@ahmedabu98 done